From 26c61059c31c60210ade4ba84be08a8a9da88124 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:13:05 -0400
Subject: [alsa] hda: add device ID for Broadwell display audio controller

Message-id: <140972823573-23-patchset-jkysela@redhat.com>
Patchwork-id: 90723
O-Subject: [RHEL 7.1 PATCH 022/442] ALSA: hda - add device ID for Broadwell display audio controller
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

This patch adds the device ID for Intel Broadwell display HD-Audio controller,
and applies Haswell properties to this device.

Signed-off-by: Mengdong Lin <mengdong.lin@intel.com>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: 862d761818ba2cb785a0a57557f97a43bd1bc922)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/hda/hda_intel.c b/sound/pci/hda/hda_intel.c
index 53937de..fa2879a 100644
--- a/sound/pci/hda/hda_intel.c
+++ b/sound/pci/hda/hda_intel.c
@@ -4009,6 +4009,9 @@ static DEFINE_PCI_DEVICE_TABLE(azx_ids) = {
    .driver_data = AZX_DRIVER_HDMI | AZX_DCAPS_INTEL_HASWELL },
  { PCI_DEVICE(0x8086, 0x0d0c),
    .driver_data = AZX_DRIVER_HDMI | AZX_DCAPS_INTEL_HASWELL },
+ /* Broadwell */
+ { PCI_DEVICE(0x8086, 0x160c),
+   .driver_data = AZX_DRIVER_HDMI | AZX_DCAPS_INTEL_HASWELL },
  /* 5 Series/3400 */
  { PCI_DEVICE(0x8086, 0x3b56),
    .driver_data = AZX_DRIVER_SCH | AZX_DCAPS_INTEL_PCH_NOPM },
-- 
1.7.1